content/doc/book/managing/ui-themes.adoc

```diff
@@ -0,0 +1,99 @@
+---
+layout: section
+title: Themes for user interface
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+It is possible to customize Jenkins' appearance with custom themes.
+This feature is not a part of the Jenkins core, but it is supported through plugins.
+There are 2 most popular plugins:
+
+* plugin:simple-theme-plugin[Simple Theme Plugin] -
+  allows customizing the Jenkins UI by providing custom CSS and Javascript files
+  It also supports replacing the Favicon.
+  There are many link:https://github.com/jenkinsci/simple-theme-plugin#themes[themes] created by Jenkins users.
+* plugin:login-theme[Login Theme Plugin] - 
+  allows modifying or replacing the Jenkins login screen.
+  It is required to customize login screens starting from Jenkins 2.128 (link:/blog/2018/06/27/new-login-page/[announcement]).
```

Comment:
is this sentence needed (2.128 is quite old now)?

typo fix though:
```suggestion
  It is possible to customize login screens starting from Jenkins 2.128 (link:/blog/2018/06/27/new-login-page/[announcement]).
```

Reply:
It was possible before (to some extent), but now it is possible only with this plugin